public/utils/utils.tsx

```diff
@@ -42,7 +42,8 @@ export const isInvalid = (name: string, form: any) =>
 export const getError = (name: string, form: any) => get(form.errors, name);
 
 export const required = (val: any): string | undefined => {
-  return !val ? 'Required' : undefined;
+  // if val is number, skip check as number value already exists
+  return typeof val !== 'number' && !val ? 'Required' : undefined;
```

Comment:
Can you verify other pages which using this method like edit feature page?

Reply:
sure. already verified.